lib/view-models/vim-normal-mode-input-element.coffee

```diff
@@ -55,7 +51,10 @@ class VimNormalModeInputElement extends HTMLDivElement
 
   removePanel: ->
     atom.workspace.getActivePane().activate()
-    @panel.destroy()
+    if @panel?
+      @panel.destroy()
+    else
+      this.parentNode.removeChild(this) if this.parentNode?
```

Comment:
I think you can avoid a conditional by using [`this.remove()`](https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove).